cluster/gce/gci/apiserver_manifest_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"strings"
 	"testing"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
nit: revert this (unnecessary)

btw - I've been seeing this change a lot lately, is there some tool that does this automatically?

Reply:
Yeah. My IDE has a google3 code format plugin, which I think automatically updates the imports to the internal format.